src/Apps/Artwork/Components/ArtworkErrorApp/ArtworkErrorApp.tsx

```diff
@@ -10,14 +10,19 @@ interface ArtworkErrorAppProps {
   artworkError: ArtworkErrorApp_artworkError$key
 }
 
+const HEADLINE_404 = "The artwork you were looking for isn't available."
```

Comment:
Please forgive me 😆 but I won't sleep tonight otherwise.

```suggestion
const HEADLINE_404 = "The artwork you were looking for isn’t available."
```

Reply:
Hahaha